src/views/default/view.php

```diff
@@ -77,6 +77,26 @@
                     }
 
                     ?>
+                    <div class="btn-group btn-group-sm" role="group">
```

Comment:
Looks like a bit too much logic for the view. Is it possible to move it to where the view is rendered and call as `$this->getPreviousRoute()`?

Reply:
Can you clarify your suggestion?

Should I create a child of `yii\web\View` and configure `yii\debug\controllers\DefaultController` for working `$this->getPreviousRoute()` or you mean something else?

I think that controllers don't need that logic, it need to be moved inside of View component, but in Yii it is not simple as it should be :)

Reply:
I prefer move logic in widget and call `NavigationButton::widget('next', $manifest, $tag)`
so view code will be:
```
<div class="btn-group btn-group-sm" role="group">
<?= NavigationButton::widget('prev', $manifest, $tag) ?>
<?= NavigationButton::widget('next', $manifest, $tag) ?>
</div>
```
@samdark what do you think?

Reply:
Yes. A widget would fit.